audio/data.py

```diff
@@ -115,15 +122,15 @@ def concat(x, y): return np.concatenate(
                 (x, y)) if len(y) > 0 else x
             
```

Comment:
Could we not do the path stuff here for all of the methods instead of each time resample etc is called? Use that list to create the new items but append new items to the old one?